build_tools/update_dependencies.sh

```diff
@@ -86,7 +86,7 @@ log_variable CLANG_BASE
 get_lib_base libgcc     11.x    platform010
 get_lib_base glibc      2.34    platform010
 get_lib_base snappy     LATEST  platform010
-get_lib_base zlib       LATEST  platform010
+get_lib_base zlib       1.2.8   platform010
```

Comment:
Question: do we need `zlib` 1.2.8 specifically with `llvm-15`?

Reply:
No. But in tp2 we only have 1.2.8 built on platform010. There is also a later version 1.2.13, but it's only built on  aarch64. Many other tp2 projects depend on zlib so upgrading it takes time. I took what we have now and we can look at upgrading zlib later.